4.7.1-windowsservercore-1709/build/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# escape=`
```

Comment:
If we're introducing a `build` and `runtime` folder, I'd like to see that consistently applied to all Dockerfiles even those without a build image right now

Reply:
I have no problems with that.